recipes/proj/6.x.x/conanfile.py

```diff
@@ -0,0 +1,96 @@
+from conans import ConanFile, CMake, tools
+import glob
+import os
+
+
+class ProjConan(ConanFile):
+    name = "proj"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://proj.org"
+    description = "Cartographic Projections and Coordinate Transformations Library"
+    topics = ("conan", "dsp", "proj", "proj4", "projections", "gis", "geospatial")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "threadsafe": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "threadsafe": True
+    }
+
+    generators = "cmake"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires.add("sqlite3/3.31.1")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["PROJ_TESTS"] = False
+        self._cmake.definitions["BUILD_LIBPROJ_SHARED"] = self.options.shared
+        self._cmake.definitions["USE_THREAD"] = self.options.threadsafe
+        self._cmake.definitions["ENABLE_LTO"] = False
```

Comment:
What is the reason for LTO disabling?

Reply:
There is no more reason to enable LTO than any other library. Let consumer decide if he wants LTO or not by providing proper linker flags himself.

Reply:
By default it's disabled, so it's ok: https://github.com/OSGeo/PROJ/blob/6.3.1/src/lib_proj.cmake#L38